### ISSUE TITLE
Expected behaviour of wake/sleep of LG TV

### ISSUE THREAD

**Issue by Aztriltus** [cmer/lg-tv-control-macos#20]
Hi, I’m wondering what should be the expected behaviour of this script when sleeping the mac. 

What I’m getting is
- Put macOS to sleep
- LG TV C3 turns black for maybe 10-15s
- TV starts showing “No signal” 
- TV goes to sleep with the fireworks animation after quite a long while, with message saying something like “Click any button except power button to wake the TV”

Then to wake the TV from “No signal” or fireworks screen
- Just move my mouse or type on keyboard to wake macOS
- TV also shows macOS interface as expected

If TV is manually turned off via the controller
- Standby light turns on
- Waking the TV from mouse/keyboard doesn’t work (which I guess should be expected) 

I think what I’m really curious about is whether there should be “No signal” after Mac goes to sleep? 

Hammerspoon logs with debug set to true seem to show that things are running ok, but I’m not 100% sure. Any help or insight is appreciated!

**Comment by devantler** [comment]
If configured correctly, the TV should shut down when you put your Mac to sleep, and turn on again when you send input that would normally wake it (clicking buttons on the keyboard or clicking your mouse)

**Comment by Aztriltus** [comment]
Interesting, thanks for letting me know. I will try to debug and see what’s up with my config. Thank you!

**Comment by devantler** [comment]
Do tell if you need some help in debugging :-) My experience is that it is a good idea to test out that `lgtv` can turn off your TV before relying on the script. And then, double-check that the script uses the configured TV name and HDMI port as well. Also, remember that Hammerspoon is a program that you need to install prior to [installing the Hammerspoon script](https://github.com/cmer/lg-tv-control-macos#installing-the-hammerspoon-script). 

You should be able to do most of this by following the `README.md`.

**Comment by Aztriltus** [comment]
Yea the "lgtv MyTV screen off ssl" command worked so I assumed everything's good. 

Anyway I managed to fix and get it working as expected now. 

Turns out the "lgtv" command is not in "~/opt" since I'm guessing I used "pyenv" to install my python. 

The "lgtv" command is actually installed in ".pyenv/shims/lgtv" 

So after updating the path, everything works as expected now. Thank you for your quick responses and offer to help. Really appreciate it! 

Happy to finally get one step closer to using TV with macOS working now!
